recipes/minio-cpp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,10 @@
+#include <miniocpp/client.h>
+
+int main(int argc, char* argv[]) {
+  minio::s3::BaseUrl base_url("play.min.io");
+
+  minio::creds::StaticProvider provider(
+      "Q3AM3UQ867SPQQA43P2F", "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG");
```

Comment:
This reads a bit too much like IDs and connections, I'd simplify the test package a lot, a simple call would be ok, thanks!

Reply:
Simplified.